admin/pre_reg/utils.py

```diff
@@ -5,10 +5,14 @@
 
 
 def get_prereg_reviewers():
-    return MyUser.objects.filter(
-        groups__name='prereg_group'
-    ).annotate(
-        fuller_name=Concat('first_name', Value(' '), 'last_name')
-    ).values_list(
-        'email', 'fuller_name'
-    )
+    # Note - fixes django.db.utils.OperationalError: no such table' error if ever one erases a db table and tries to remigrate
+    try:
+        return MyUser.objects.filter(
+            groups__name='prereg_group'
+        ).annotate(
+            fuller_name=Concat('first_name', Value(' '), 'last_name')
+        ).values_list(
+            'email', 'fuller_name'
+        )
+    except Exception:
```

Comment:
Even if this error handling is legitimate, bare `except`s are bad because they hide legitimate errors.